File: exp/UnityChan2DTutorial/Assets/ScriptsJs/UI/PressX.js
#pragma strict

function Start () {
  InvokeRepeating("ShowText", 0, 0.5) ;
}

function ShowText(){
  var t:UI.Text = GetComponent.<UI.Text>() ;
  if(t.text == ""){
    t.text = "PRESS X" ;
  }else{
    t.text = "" ;
  }
}

File: exp/UnityChan2DTutorial/Assets/ScriptsCSharp/UI/PressX.cs
using UnityEngine;
using UnityEngine.UI ;
using System.Collections;

public class PressX : MonoBehaviour {

	void Start () {
		InvokeRepeating("ShowText", 0, 0.5f) ;
	}
	
	void ShowText () {
		Text t = GetComponent<Text>() ;
		if(t.text == ""){
			t.text = "PRESS X" ;
		}else{
			t.text = "" ;
		}
	}
}
